### PR TITLE
refactor refresh of wireguard keys

### PR DESCRIPTION
**Pull request by mattkasun**
## Describe your changes
key updates are handled in host updates with updateaction = UpdateKeys
## Provide testing steps
see GRA-1529

## Checklist before requesting a review
- [x] My changes affect only 10 files or less.
- [x] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [x] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [ ] Netclient & Netmaker are awesome.

### CODE REVIEW

File: functions/daemon.go
Comment (0xdcarns) [inline functions/daemon.go:400]: debug log

File: functions/daemon.go
Comment (0xdcarns) [inline functions/daemon.go:406]: debug log

File: functions/daemon.go
Comment (0xdcarns) [inline functions/daemon.go:413]: debug log

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

File: functions/daemon.go
Comment (mattkasun) [reply functions/daemon.go]: debug logs removed

**Review by theguy951357 — APPROVED** [review]
pr passes
critical list passes

**Review by Aceix — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
pr steps pass
critical list passes

File: functions/mqhandlers.go
Comment (abhishek9686) [inline functions/mqhandlers.go:231]: why is it required to pass mq client as parameter to `UpdateKeys`

**Comment by mattkasun** [comment]
depends on netmaker PR -- 2199
do not merge until 2199 is merged and go.mod updated

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**
